examples/megacli.cpp

```diff
@@ -22,6 +22,7 @@
 #include "mega.h"
 #include "megacli.h"
 #include <fstream>
+#include <mega/autocomplete.h>
```

Comment:
this breaks linux compilation in QT if configured to compile with c++11 (autocomplete.cpp is not added)

Reply:
Good point.  I have added autocomplete.cpp to the various project files in the SDK, in case there are any other similar cases.  It's good to have it there for consistency, and will only be compiled if the compiler is configured for C++11 or later.